resources/assets/uad_lists.json

```
@@ -37236,5 +37236,59 @@
     "dependencies": [],
     "neededBy": [],
     "labels": []
+  },
+  {
+    "id": "com.android.DeviceAsWebcam",
+    "list": "Oem",
+    "description": "Allows using certain Pixel phones as a USB webcam.\n Will break USB function picker.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.android.safetyregulatoryinfo",
+    "list": "Aosp",
+    "description": "Shows regulatory and general safety information in Settings.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.android.settings.husky",
+    "list": "Oem",
+    "description": "Settings specific to Google Pixel 8 Pro hardware.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.google.android.aicore",
+    "list": "Google",
+    "description": "Required for most AI features on Google Pixel devices such as magic eraser, best take, now playing etc.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.google.android.apps.diagnosticstool",
+    "list": "Oem",
+    "description": "Google Pixel diagnostics tool.\n Exclusive to Pixel Watch 1 & 2 along with Pixel 6 series and later running Wear OS 4/Android 14.",
```

```suggestion
    "description": "Google Pixel diagnostics tool.\nExclusive to Pixel Watch 1 & 2 along with Pixel 6 series and later running Wear OS 4/Android 14.",
```
Remove redundant space.